packages/calling-component-bindings/package.json

```diff
@@ -36,12 +36,12 @@
     "reselect": "~4.0.0"
   },
   "peerDependencies": {
-    "@azure/communication-calling": "1.4.3-beta.1 || 1.3.2",
+    "@azure/communication-calling": "1.4.3-beta.1 || 1.4.3",
```

Comment:
1.4.3-beta.1 is behind 1.4.3, should this just be  
`"@azure/communication-calling": "1.4.3",`

or to make it simpler when we upgrade again
`"@azure/communication-calling": "1.4.3 || 1.4.3",` lol

Reply:
Yeah, it wouldn't work. Our tooling for changing the version for release branches is primitive. Also, @azure/communication-calling doesn't follow SemVer properly for beta packages (`1.4.3-beta.1` has lots of things not in `1.4.3`) so it is more "correct" to explicitly state that both are allowed.

Reply:
Oh yeah good point it'll have beta functionality